src/azure-cli-core/azure/cli/core/aaz/_field_value.py

```diff
@@ -444,3 +445,113 @@ def to_serialized_data(self, processor=None, keep_undefined_in_list=False,  # py
         if processor:
             result = processor(self._schema, result)
         return result
+
+
+class AAZIdentityObject(AAZObject):
+    M_USER_ASSIGNED = "miUserAssigned"
+    S_USER_ASSIGNED = "userAssigned"
+    M_SYSTEM_ASSIGNED = "miSystemAssigned"
+    S_SYSTEM_ASSIGNED = "systemAssigned"
```

Comment:
Please move those const values out of the class. https://github.com/Azure/azure-cli/blob/414043d8f12fcec26ab9a191db2b28caba460d17/src/azure-cli-core/azure/cli/core/aaz/_field_value.py#L86-L90